apps/builder/app/builder/features/breakpoints/breakpoints-selector.tsx

```diff
@@ -18,11 +18,14 @@ import { groupBreakpoints } from "~/shared/breakpoints";
 export const BreakpointsSelector = () => {
   const breakpoints = useStore(breakpointsStore);
   const selectedBreakpoint = useStore(selectedBreakpointStore);
-  const ref = useRef(null);
+  const refs = useRef(new Map<string, HTMLButtonElement>());
```

Comment:
why do you need a map, seems like it could be a normal ref

Reply:
https://react.dev/learn/manipulating-the-dom-with-refs#how-to-manage-a-list-of-refs-using-a-ref-callback

Reply:
oh you want many nodes

Reply:
It can't be a single ref, as the issue was that the ref was outdated. And not set